.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
do you know what made this get added?

Here is some context:

![image](https://github.com/frc1675/frc1675-2024-general/assets/282467/6af30890-39b7-4d57-b517-68463ae0a1c7)


I think usually our (windows) computers should have autocrlf and this doesn't get explicitly set. i'm not saying delete it as I dont think it hurts as long as windows users have autocrlf on but I am curious if something automatically put this here

Reply:
(Maybe we WANT this explicitly set on our repos for the linux users? do we know if linux users have had any hangups?)

Reply:
Oh sorry I meant to mention this explicitly in the PR but forgot. I have had hangups and that is how I was able to resolve it. Git was showing files as modified when they were not because the line ending had changed. I have the repo on my Linux laptop and windows desktop and both are now working fine. I think that there should not be issues with merging that.

Reply:
The only thing we may have to do is set core.autocrlf to false on windows machines, I did do that on my windows machine.

Reply:
We know from Saturday that you want your autocrlf to true on windows, but we should actually put this text = auto in our repos to ensure all of them stay normalized.